python/CreatePipWhl.cmake

```diff
@@ -53,6 +53,30 @@ if (PYTHON)
         # 1. Get the NGRAPH_VERSION exported from nGraph CMake
         # 2. If #1 isn't possible, then determine this by looking at the 
         # dependency list?
+        execute_process(COMMAND 
```

Comment:
Looks like this command is repeated below (just past the greentext).

Reply:
Yes, with different arguments. :|

Reply:
Ooh, there it is. `libcpu_backend` vs `libngraph_device`. Okey dokey.